public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -31,7 +31,6 @@ import {
   SupplementaryQueryOptions,
   toUtc,
   AnnotationEvent,
-  FieldType,
```

Comment:
Aren't all of these dependecies going to be removed with https://github.com/grafana/grafana/pull/80281? I think change in this file should be reverted as you have already PR that tackles it.